### PR TITLE
Update to work with v0.18.12

### PR DESCRIPTION
**Pull request by elanqo**
@datajoely I was interested in using intake, in particular intake-dremio and intake-salesforce with Kedro.
Toward that, I thought I would start by making your prototype work again with the latest Kedro. 

I see that while initially, Kedro had integration with Anaconda Intake as an idea and good first issue - it has not been pursued since. 

If you some guidance and ideas, I would be happy to do the work required and contribute back to your repo / Kedro

- Upgrade to leverage kedro-datasets plugin
- Populating a Kedro Catalog from an Intake one.
- The reverse: Populating an Intake catalog from a Kedro catalog.yml

Thanks, and Regards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Pre-requisite
+This is a kedro project, prior to running install kedro, then use kedro command to run the project.
+
+    pip install kedro~=0.18.1
```
Comment (datajoely) [inline README.md]:
```suggestion
    pip install kedro~=0.18.12
```

File: pyproject.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [tool.kedro]
 package_name = "space"
 project_name = "space"
-project_version = "0.17.5"
+project_version = "0.18.1"
```
Comment (datajoely) [inline pyproject.toml]:
```suggestion
project_version = "0.18.12"
```

**Review by datajoely — COMMENTED** [review]
Thank you for this! Hi @elanqo so I paused this since I couldn't get many people interested it at the time! Going forward I'd love to make Kedro better integrate with Intake + things like [Ibis](https://ibis-project.org/) so that users would benefit from those connector ecosystems for free.

**Review by datajoely — APPROVED** [review]

File: src/requirements.txt [hunk 3]
```diff
@@ -1,455 +1,18 @@
...
-# setuptools
+black~=22.0
+flake8>=3.7.9, <5.0
+ipython>=7.31.1, <8.0; python_version < '3.8'
+ipython~=8.10; python_version >= '3.8'
+isort~=5.0
+jupyter~=1.0
+jupyterlab_server>=2.11.1, <2.16.0
+jupyterlab~=3.0, <3.6.0
+kedro~=0.18.11
```
Comment (rxm7706) [inline src/requirements.txt:9]: can change kedro~=0.18.11 to kedro>=0.18.12 
to match the readme --     pip install kedro~=0.18.12

**Comment by rxm7706** [comment]
> Thank you for this! Hi @elanqo so I paused this since I couldn't get many people interested it at the time! Going forward I'd love to make Kedro better integrate with Intake + things like [Ibis](https://ibis-project.org/) so that users would benefit from those connector ecosystems for free.

+ 1 for intake & ibis... Just to give a high level idea of what that would do in terms of connector ecosystems... 
# ----------------------
# ----------------------
  - ibis-framework >=6.0.0
  - ibis-bigquery >=6.0.0
  - ibis-dask >=6.0.0
  - ibis-datafusion >=6.0.0
  - ibis-duckdb >=6.0.0
  - ibis-mssql >=6.0.0
  - ibis-mysql >=6.0.0
  - ibis-oracle >=6.0.0
  - ibis-polars >=6.0.0
  - ibis-postgres >=6.0.0
  - ibis-pyspark >=6.0.0
  - ibis-sqlite >=6.0.0
# ----------------------
# ----------------------
  - intake >=0.7.0
  - intake-dataframe-catalog >=0.2.2
  - intake-dremio >=0.2.3 
  - intake-elasticsearch >=0.2.0
  - intake-excel >=0.1.4
  - intake-odbc >=0.1.0
  - intake-parquet >=0.3.0
  - intake-pattern-catalog >=2023.3.0
  - intake-salesforce >=0.0.10
  - intake-spark >=0.1.2
  - intake-sql >=0.4.0
  - intake-streamz >=0.0.1

**Pull request closed without merging**